test/integration/specs/kbaseCellToolbarMenu_data.json

```diff
@@ -0,0 +1,165 @@
+{
+    "common": {
+        "unselectedBorder": "1px 1px 1px 5px solid rgb(204, 204, 204)",
+        "selectedBorder": "1px 1px 1px 5px solid rgb(75, 184, 86)",
+        "cells": [
+            {
+                "title": "Narrative Cell Toolbar Testing"
+            },
+            {
+                "title": "Cell 1"
+            },
+            {
+                "title": "Cell 2"
+            },
+            {
+                "body": "Cell 3"
+            },
+            {
+                "body": "Cell 4"
+            }
+        ]
```

Comment:
this information already seems to be in the `cases` object -- does it need to be repeated here?